memif-attach/main.go

```diff
@@ -18,31 +18,62 @@ package main
 
 import (
 	"context"
+	"net"
 	"time"
-
+	// "io"
 	"git.fd.io/govpp.git/api"
 	"github.com/edwarnicke/govpp/binapi/memif"
+	interfaces "github.com/edwarnicke/govpp/binapi/interface"
+	//  "github.com/edwarnicke/govpp/binapi/interface_types"
 	"github.com/edwarnicke/log"
 	"github.com/edwarnicke/vpphelper"
 )
 
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
-
+	
 	// Connect to VPP with a 1 second timeout
 	connectCtx, _ := context.WithTimeout(ctx, time.Second)
 	conn, vppErrCh := vpphelper.StartAndDialContext(connectCtx)
 	exitOnErrCh(ctx, cancel, vppErrCh)
-
+	// Set AdminUp
+	swinterface := interfaces.SwInterfaceSetFlags {
+		SwIfIndex: 1, 
+		Flags: 1,  
```

Comment:
Recommend using https://github.com/edwarnicke/govpp/blob/main/binapi/interface_types/interface_types.ba.go#L31 for Flags